python/llm/example/GPU/README.md

```diff
@@ -0,0 +1,7 @@
+# BigDL-LLM Examples on Intel GPU
```

Comment:
Also need to document the GPU supported and the configurations

Reply:
> Also need to document the GPU supported and the configurations

Have added some descriptions about CPU/GPU supported in `python/llm/example/CPU/README.md` and `python/llm/example/GPU/README.md`.